e2e-tests/src/main/java/io/kafbat/ui/screens/topics/TopicSettingsTab.java

```diff
@@ -36,7 +39,25 @@ private TopicSettingsTab.SettingsGridItem getItemByKey(String key) {
 
   @Step
   public String getValueByKey(String key) {
-    return getItemByKey(key).getValue();
+    while (true) {
```

Comment:
When the key is not in the first page, we need to navigate page by page to find the value